README.md

```diff
@@ -4,7 +4,7 @@ Welcome! This repository houses all of the assets required to build the Kubernet
 
 ## Contributing to the docs
 
-You can click the **Fork** button in the upper-right area of the screen to create a copy of this repository in your GitHub account. This copy is called a *fork*. Make any changes you want in your fork, and when you are ready to send those changes to us, go to your fork and create a new pull request to let us know about it.
+You can click the **Fork** button in the upper-right area of the screen to create a copy of this repository into your GitHub account. This copy is called a *fork*. Make any changes you want in your fork, and when you are ready to send those changes to us, go to your fork and create a new pull request to let us know about it.
```

Comment:
I suppose this might be technically more accurate than the existing text, because your fork lives within your Github account's "organization".

Reply:
Deep nit territory:

Because "in" and "into" are more or less interchangeable ("into" an organization that's "in" your account), I wonder if "for" might be more helpful here:

> You can click the **Fork** button in the upper-right area of the screen to create a copy of this repository for your GitHub account.

In the absence of a clearly preferable alternative (they all seem interchangeable to me), let's keep the existing text.